New IT blogpost added

tuncK opened the pull request:
Upon Sanjay's suggestion, I tried to write a new blogpost about interactive tools.

sanjaysrikakulam commented on content/blog/2023-08-15-ITs/index.md:
```diff
@@ -0,0 +1,60 @@
+
+## What are interactive tools (IT)?
+
+For a non-interactive tool execution, the user defines all inputs before the job execution and has access to the results after the job finishes. During the execution phase of the job, the user does not and cannot interact with the system. For majority of the scientific software, this model is actually sufficient, because they are commandline-only, the computation follows a pre-written recipe in t ...
```
I am not sure this constitutes the only role of ITs. This is just an extra option/luxury that the users get.

For me IT's or VREs are more like the below:

ITs or VREs (Virtual research environments) provide researchers with digital spaces to conduct experiments, analyze data, and collaborate seamlessly. VREs such as Jupyter Notebook and Rstudio empower users with interactive coding environments, enabling real-time manipulation of code, visualizations, and data analysis and fostering iterative exploration. Thus, enhancing efficiency and reproducibility within these dynamic research landscapes.

sanjaysrikakulam commented:
I think it should be a news item. You might have to move it from `blog` to `news`.

tuncK commented:
> I think it should be a news item. You might have to move it from `blog` to `news`.

Moved as you suggested.

tuncK commented on content/blog/2023-08-15-ITs/index.md:
```diff
@@ -0,0 +1,60 @@
+
+## Updates from usegalaxy.eu
+
+Within the four-year time window between September 2019 and August 2023, our public instance, [live.usegalaxy.eu](https://live.usegalaxy.eu), already had served 21 801 IT jobs executions. A coarse look into the job statistics suggests that the typical availability of the system was not very different than that of a user-owned local system. Thanks to the availability of VMs dedicated to IT executi ...
```
As I expressed during our conversation before, unless @bgruening overrules: 

I want to keep the rightmost column in the graph (and a short argument explaining it to the attentive readers). Without backing with some orthogonal data, I think the outliers are part of a distribution. Transient glitches are part of our imperfect service quality. It functions pretty good nowadays, just that it did cause to some users at some point.

tuncK replied:
Updated the figure and explanation.

tuncK commented:
Hi @sanjaysrikakulam, as we discussed on Tuesday, I added sectioning, changed a few wordings and added the "steps to launch an IT" section as I mentioned. Any other issues that you notice?

sanjaysrikakulam reviewed (approved):
It looks great after the last changes. Thank you!

bgruening reviewed (approved):
Thanks @tuncK
